connection/testConnection.js

```diff
@@ -9,13 +9,12 @@ const tu = require('../utils/testUtils');
 describe('connection tests >', () => {
   before(tu.buildConnection);
 
-  const ctx = {
-    baseTrustUrl: 'https://trust.salesforce.com/api',
-    statusLinkUrl: 'http://status.salesforce.com/status',
-  };
-
   describe('prepareUrl >', () => {
     it('prepareUrl, default window', () => {
+      const ctx = {
```

Comment:
Test was failing on master, one test was affecting the other (context was incorrect bcz aspect was not defined).
Duplicated the ctx and it works, didnt spent time to that.